views/layouts/main.njk

```diff
@@ -15,6 +15,28 @@
             </ul>
         </nav>
 
+        <div class="modal js-modal u-hidden">
```

Comment:
Do we have to meet the progressive enhancement & accessibility needs ?

As mentioned in this guidance - https://docs.google.com/document/d/16BEBhmodDoUyIeyVAfyj3rMX45Lw0Zo5rs0pWC3_5Mw/edit.

Reply:
Great point – this would definitely need some attention before we launch it. I think we can merge this as-is now but if/when we decide to *use* the modal (we might not need to!) we should look at this.